content/rs/references/cli-utilities/crdb-cli/_index.md

```diff
@@ -36,8 +36,11 @@ If you sign in with a non-root user, you must add `/opt/redislabs/bin/` to your
 - Add or remove an instance of the Active-Active database on a specific cluster.
 
 Each command creates a task.
+
 By default, the command runs immediately and displays the result in the output.
+
 If you use the <nobr>`--no-wait`</nobr> flag, the command runs in the background so that your application is not delayed by the response.
-You can track the task with [`crdb-cli task status`]({{<relref "/rs/references/cli-utilities/crdb-cli/task-status">}}).
+
+Use the [`crdb-cli task` commands]({{<relref "/rs/references/cli-utilities/crdb-cli/task/">}}) to manage Active-Active database tasks.
```

Comment:
I think most customers won't really know how to use this correctly. Can we add some note indicating its for expect users (or by guidance of support)?

Reply:
Note added.